pkg/prm/prm_test.go

```diff
@@ -0,0 +1,647 @@
+package prm_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/puppetlabs/prm/pkg/prm"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	// hide logging output
+	log.Logger = zerolog.New(ioutil.Discard).With().Timestamp().Logger()
+	os.Exit(m.Run())
+}
+
+func TestGet(t *testing.T) {
+	type args struct {
+		toolDirPath string
+		setup       bool
+		toolConfig  string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    prm.Tool
+		wantErr bool
```

Comment:
One pattern I think we should get out of as a team is just checking for the presence of the error and not validating the message coming back.